build.gradle

```diff
@@ -160,16 +162,16 @@ subprojects {
         }
 
         tasks.withType(KotlinCompile).configureEach {
-            kotlinOptions {
+            compilerOptions {
                 allWarningsAsErrors = true
-                languageVersion = '1.7'
-                apiVersion = '1.7'
+                languageVersion = KOTLIN_1_8
+                apiVersion = KOTLIN_1_8
                 verbose = true
-                jvmTarget = javaVersion
-                freeCompilerArgs += [
-                    "-Xjvm-default=all",
-                    "-java-parameters"
-                ]
+                jvmTarget = JVM_11
```

Comment:
I assume that `javaVersion` (used in the previous version of this file) is a variable. What happened to it?

Reply:
> I assume that `javaVersion` (used in the previous version of this file) is a variable. What happened to it?

It still exists, but is not assignable to the `jvmTarget` option in the Kotlin plugin's new DSL :facepalm:.

Reply:
So do we still use it elsewhere, and now need to make sure things are in sync? Or can it be deleted?

Reply:
> So do we still use it elsewhere, and now need to make sure things are in sync? Or can it be deleted?

We use it elsewhere. This isn't ideal but it is good enough for now.